src/commands/lint/configureLintingToolsCommand.ts

```diff
@@ -64,7 +81,7 @@ export class ConfigureLintingToolsCommand {
 
             // Ask user to add eslint plugin
             const result = await this.showMessage(
```

Comment:
I wonder: do we want to make this all or nothing, now that we're adding multiple plugins? Thinking about it, I guess we can start here, and we can always change it to ask for each plugin, if we get user pushback. KISS to start, works for me.

Reply:
or a window with three checkboxes instead of asking three times? should be possible I guess, not sure about effort yet as newbie.

Reply:
Let's do it in one shot like you are here. We can always get fancy if/when we run into scenarios where users want the granularity.

Reply:
(There are a number of different ways, but there's also a fair amount of UX corner case consideration to implement them, so I'd rather not jump all the way into that deep end.)

Reply:
oh, just thought a bit more using checkboxes will that it will more complex due to user could launch the wizard again and then we need to mark the checkbox checked,  ...  for sure more work.  based on no complains from users even wizard is broken for a while, I will suggest to go easy,  just one msg for 3 eslint plugins.